client/app/models/BillingService.class.js

```diff
@@ -6,6 +6,7 @@ export default class BillingService {
     Object.assign(this, service);
 
     this.expirationDate = moment(this.expiration);
+    this.state = this.isSuspended() ? 'EXPIRED' : 'UP';
```

Comment:
This is weird. Why can I `service.isSuspended()` but I can't `service.isUp()` ?

Reply:
Could do but `serviceisUp() === !service.isSuspended()`.